content/swift/concepts/arrays/terms/allSatisfy/allSatisfy.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: '.allSatisfy()'
+Description: 'Returns a Boolean expression indicating whether every element of a sequence satisfies a given condition.'
+Subjects:
+  - 'IOS'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'SwiftUI'
+CatalogContent:
+  - 'learn-Swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+** .allSatisfy() ** is a Swift method that checks if all the elements in an array satisfies a given condition. 
+
+## Syntax
+```pseudo
+array.allSatisfy(condition)
```

Comment:
We need a sentence here explaining what `condition` is, and what can be put there.

Reply:
Sentence added, please let me know if changes are needed